main.py

```diff
@@ -133,6 +133,18 @@ def get_jenkins_jobs(server, job_search_fields):
 
 	return relevant_jobs
 
+def get_bugs_set(blockers_file):
+#Generates a set from the entire list of bugs
+	bug_list=[]
+	for job in blockers_file:
+		bz = blockers_file[job]['bz']
+		if bz == [0]:
```

Comment:
Can you explain what this line does?

Reply:
When the bz is 0, means there is nothing on the blockers list for that job, it shouldn't append the 0 to the list of bugs so it resumes iterating over the next items.